### PR TITLE
Update bee to d49ba172e644

### PR DESCRIPTION
**Pull request by manugildev**
NDK r27b toolchain has been recently added to Bee.
This PR updates Bee to latest available version to allow us to access newer NDK in preparation for future work.

For reference, last update of Bee in android-jni-bridge was on Feb 08th, 2023.

### CODE REVIEW

File: JniBridge.bee.cs [hunk 1]
```diff
@@ -196,6 +201,26 @@ static Jdk SetupJava()
         return new Jdk(openJdk.Path.ResolveWithFileSystem());
     }
 
+    private static bool IsValidJdkVersion(string javaHomePath, out string version)
+    {
+        var versionFilePath = Path.Combine(javaHomePath, "version.txt");
```
Comment (todi1856) [inline JniBridge.bee.cs]: Shouldn't you be reading java version from release file, which looks like this:
```
IMPLEMENTOR="Eclipse Adoptium"
IMPLEMENTOR_VERSION="Temurin-17.0.9+9"
JAVA_RUNTIME_VERSION="17.0.9+9"
JAVA_VERSION="17.0.9"
JAVA_VERSION_DATE="2023-10-17"
LIBC="default"
MODULES="java.base java.compiler java.datatransfer java.xml java.prefs java.desktop java.instrument java.logging java.management java.security.sasl java.naming java.rmi java.management.rmi java.net.http java.scripting java.security.jgss java.transaction.xa java.sql java.sql.rowset java.xml.crypto java.se java.smartcardio jdk.accessibility jdk.internal.jvmstat jdk.attach jdk.charsets jdk.compiler jdk.crypto.ec jdk.crypto.cryptoki jdk.crypto.mscapi jdk.dynalink jdk.internal.ed jdk.editpad jdk.hotspot.agent jdk.httpserver jdk.incubator.foreign jdk.incubator.vector jdk.internal.le jdk.internal.opt jdk.internal.vm.ci jdk.internal.vm.compiler jdk.internal.vm.compiler.management jdk.jartool jdk.javadoc jdk.jcmd jdk.management jdk.management.agent jdk.jconsole jdk.jdeps jdk.jdwp.agent jdk.jdi jdk.jfr jdk.jlink jdk.jpackage jdk.jshell jdk.jsobject jdk.jstatd jdk.localedata jdk.management.jfr jdk.naming.dns jdk.naming.rmi jdk.net jdk.nio.mapmode jdk.random jdk.sctp jdk.security.auth jdk.security.jgss jdk.unsupported jdk.unsupported.desktop jdk.xml.dom jdk.zipfs"
OS_ARCH="x86_64"
OS_NAME="Windows"
SOURCE=".:git:a4bbf40198b1"
BUILD_SOURCE="git:16ac032d56eb0820393ca7752c13a4bab98eaf92"
BUILD_SOURCE_REPO="https://github.com/adoptium/temurin-build.git"
SOURCE_REPO="https://github.com/adoptium/jdk17u.git"
FULL_VERSION="17.0.9+9"
SEMANTIC_VERSION="17.0.9+9"
BUILD_INFO="OS: Windows Server 2022 Version: 10.0"
JVM_VARIANT="Hotspot"
JVM_VERSION="17.0.9+9"
IMAGE_TYPE="JDK"
```

I assume version.txt, is something bee creates, reading jdk file would be more correct.

Also, could you please use regex to parse version

**Review by aurimasc — APPROVED** [review]
Java version check aside I see no issues with this.

File: JniBridge.bee.cs (hunk 1 above)
Comment (manugildev) [reply JniBridge.bee.cs]: I have just checked, some of the JAVA installations I have on my machine (from Android Studio mainly) don't have a `release` file, some don't have a `version.txt`.

Seems like the most reliable way is to run `java -version` and parse the output.

Comment (todi1856) [reply JniBridge.bee.cs]: I see, whatever works best for you.

**Review by todi1856 — APPROVED** [review]

File: JniBridge.bee.cs (hunk 1 above)
Comment (manugildev) [reply JniBridge.bee.cs]: 8597c7f

**Pull request merged**